packages/workflow-runtime/package.json

```
@@ -38,6 +38,12 @@
     "typescript": "4.8.4"
   },
   "dependencies": {
+    "@opentelemetry/api": "^1.2.0",
+    "@opentelemetry/exporter-trace-otlp-http": "^0.33.0",
+    "@opentelemetry/resources": "^1.7.0",
+    "@opentelemetry/sdk-node": "^0.33.0",
+    "@opentelemetry/semantic-conventions": "^1.7.0",
+    "@temporalio/interceptors-opentelemetry": "^1.4.0",
```

Pin these dependencies. Renovate will otherwise open a follow-up PR to do that.

```suggestion
    "@temporalio/interceptors-opentelemetry": "1.4.0",
```

---

Fixed.